tests_integration/test_streamlit.py

```diff
@@ -9,49 +18,53 @@
 )
 
 
+@pytest.mark.skipif(
+    sys.platform == "win32",
+    reason="Skip for windows, Path.is_mount() is unsupported on this system",
```

Comment:
Do we have a task to do sth with this?

Reply:
NADE team is aware of it

Reply:
Ok, but isn't it making our existing commands unusable on Windows? It's becoming a problem not only for `snow app` but also for other commands, like `snow streamlit` in this case.

Reply:
Right, maybe we should wait with merge until this will be fixed

Reply:
I've added a fix to this branch: https://github.com/Snowflake-Labs/snowcli/pulls